### ISSUE TITLE
- ERROR API network error

### ISSUE THREAD

**Issue by gh-jslee1301** [keybase/keybase-issues#2885]
Post https://api.keybase.io/_/api/1.0/logdump/send.json: dial tcp: lookup api.keybase.io: getaddrinfow: No such host is known.

**Comment by maxtaco** [comment]
are you have DNS issues?

**Comment by gh-jslee1301** [comment]
yes , i do.. my notebook can online but browser always offline.few days cant fixed it.
